linux/mcompile/keymap/keymap.cpp

```diff
@@ -15,15 +15,25 @@ int convert_Shiftstate_to_LinuxShiftstate(int shiftState) {
   else return shiftState;                                                                                            // Lin ss x  -> Lin ss x
 }
 
+int convert_rgkey_Shiftstate_to_LinuxShiftstate(int shiftState) {
```

Comment:
I think we should use `ShiftState` as type for the parameter:

```suggestion
int convert_rgkey_Shiftstate_to_LinuxShiftstate(ShiftState shiftState) {
```

And then below we can do:

```
if (shiftState == Base) return 0;
else if (shiftState == Shft) ...
```